ietfdata/datatracker.py

```diff
@@ -1338,28 +1338,27 @@ def meeting_types(self) -> Iterator[MeetingType]:
 
     def related_documents(self, 
         source               : Optional[Document]         = None, 
-        target               : Optional[Document]         = None, 
+        target               : Optional[DocumentAlias]    = None, 
```

Comment:
It's not clear why `source` and `target` have different types?

Reply:
![related document](https://user-images.githubusercontent.com/35597579/78332609-427d6800-7580-11ea-9eee-3f38490345e9.PNG)


A related document has a document type as source and docalias as its target.